.gitignore

```diff
@@ -1,2 +1,2 @@
 draughtsman
-
+TODO
```

Comment:
You will notice that a lot of projects have this in the gitignore. I use it for my personal workflow to note intraday TODOs per project.